playbooks/aws/openshift-cluster/build_ami.yml

```diff
@@ -35,6 +35,8 @@
 # to become a node.
 - import_playbook: ../../openshift-node/private/image_prep.yml
 
+- import_playbook: reboot_instance.yml
```

Comment:
I think it is worthwhile adding a comment for why we need to do this. "Some updates are only fully baked after a reboot" - otherwise it might be tricky to track down again if this was removed.

Reply:
Updated